app/js/views/app_view.js

```diff
@@ -1,16 +1,43 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'text!templates/app_view.ejs',
+  'collections/todo'
+], function ($, Backbone, indexTemplate, todoCollection) {
   return Backbone.View.extend({
     el: $('#todo-app'),
+    KEY_ENTER_CODE: 13,
 
-    currentDate: function () {
-      return new Date();
+    events: {
+      'click #addButton': 'addItemsHandler',
```

Comment:
Why add item__s__  you only add 1 todo at a time.

Also use the domain language, we are adding todos not times.